linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/IstioIdentifier.scala

```diff
@@ -23,14 +24,36 @@ class IstioIdentifier(pfx: Path, baseDtab: () => Dtab, routeCache: RouteCache, c
     }
   }
 
+  def compareStringMatch(headerValue: String, stringMatch: StringMatch): Boolean = {
```

Comment:
TIOLI: naming this something like `headerMatches` would make it a bit more clear to me that it returns true for a match and false if it does not match.